ui/components/app/configure-snap-popup/configure-snap-popup.tsx

```diff
@@ -6,10 +6,10 @@ import {
   Box,
   Modal,
   ModalOverlay,
-  ModalContent,
-  ModalHeader,
   Text,
 } from '../../component-library';
+import { ModalContent } from '../../component-library/modal-content/deprecated';
+import { ModalHeader } from '../../component-library/modal-header/deprecated';
 import { useI18nContext } from '../../../hooks/useI18nContext';
 import {
   AlignItems,
```

Comment:
Screenshots showing no visual changes to UI

### Before
<img width="1512" alt="Screenshot 2023-12-10 at 2 33 09 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/d2ae0ed5-1344-4cc3-a708-0ec032be5c9e">

### After
<img width="1512" alt="Screenshot 2023-12-10 at 2 33 22 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/c9e9c80b-13bb-4760-aac5-fe75267789bf">